### PR TITLE
Persists inputs in a prompt in playground in localStorage

### PR DESCRIPTION
**Pull request by andresgutgon**
## What? 
Persist inputs by commit version UUID and document UUID

### CODE REVIEW

File: apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/Playground/index.tsx [hunk 1]
```diff
@@ -75,24 +69,11 @@ export default function Playground({
   metadata: ConversationMetadata
 }) {
   const [mode, setMode] = useState<'preview' | 'chat'>('preview')
-  const [inputs, setInputs] = useState<Record<string, string>>({})
-  const parameters = useMemo(() => convertParams(inputs), [inputs])
-
-  useEffect(() => {
```
Comment (andresgutgon) [inline apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/Playground/index.tsx:81]: One less `useEffect`. I'm making the world better

File: apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/Playground/index.tsx [hunk 2]
```diff
@@ -1,47 +1,41 @@
...
     },
-    [inputs],
+    [inputs, setInputs],
   )
+  useEffect(() => {
+    setSsr(false)
+  }, [])
+
+  // Avoid rendering inputs on server
+  // We have a Hydration issue with the inputs becasue
```
Comment (geclos) [inline apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/Playground/index.tsx:36]: typo becasue

Comment (geclos) [reply apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/Playground/index.tsx:36]: but i don't get it this has the 'use client' pragma

Comment (andresgutgon) [reply apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/Playground/index.tsx:36]: Client components are also rendered on the server. Client pragma means react will hydrate this component

**Pull request merged**